packages/react-hooks/README.md

```diff
@@ -140,3 +140,47 @@ const Example = React.forwardRef(function Example(props:{}, forwardedRef: React.
   return <div>Example</div>;
 })
 ```
+
+## useSetInterval
+
+`function useSetInterval()`
+
+Hook that returns a wrapper function for `setInterval` which automatically handles disposal.
+
+### Example
+
+```jsx
+import { useSetInterval } from '@uifabric/react-hooks';
+
+const MyComponent = buttonContent => {
```

Comment:
For demo purposes it's probably best to just get rid of the props and hard-code a string inside the button (same in useSetTimeout)